src/System.CommandLine/Argument.cs

```diff
@@ -19,24 +19,10 @@ public abstract class Argument : Symbol, IValueDescriptor
         private List<Func<CompletionContext, IEnumerable<CompletionItem>>>? _completionSources = null;
         private List<Action<ArgumentResult>>? _validators = null;
 
-        /// <summary>
-        /// Initializes a new instance of the Argument class.
-        /// </summary>
-        protected Argument()
+        private protected Argument(string name) : base(name, canContainWhitespaces: true)
```

Comment:
since arguments name is not used for parsing, I've decided to keep allowing the whitespaces. I don't have a strong opinion about that.

Reply:
I agree.